jsapp/js/components/processing/singleProcessingPreview.module.scss

```diff
@@ -3,7 +3,7 @@
 @use './singleProcessingVariables';
 
 .root {
-  margin-top: singleProcessingVariables.$s-single-processing-tabs;
+  margin-top: sizes.$x32;
```

Comment:
I see that this needs some work. Let's try to remember to somehow use that variable here